sdk/keyvault/keyvault-certificates/recordings/browsers/certificates_client__create_read_update_and_delete/recording_can_delete_a_certificate.json

```diff
@@ -13,7 +13,7 @@
     "cache-control": "no-cache",
     "content-length": "87",
     "content-type": "application/json; charset=utf-8",
-    "date": "Sat, 07 Mar 2020 16:14:52 GMT",
+    "date": "Wed, 01 Apr 2020 20:16:00 GMT",
```

Comment:
Why does the recording need an update? We aren't changing the actual outgoing request. We are only repacking retries with a while loop right?

Reply:
yes, but the number of requests changes most likely. I decided to re-record them to stay clear with it.

Reply:
Why would the number of requests make any difference to the recording? I would imagine that we would have a single recorded request

Reply:
This file has thousands of new additions with too many newly recorded requests with errors as responses.
@sadasant, please double-check if something unexpected is happening here.